config/default/core.extension.yml

```diff
@@ -121,6 +121,7 @@ module:
   token: 0
   toolbar: 0
   tour: 0
+  twig_tweak: 0
```

Comment:
We will probably not want to enable this for all sites unless it is actually necessary with the updates we are making.